dcpy/connectors/edm/publishing.py

```diff
@@ -24,14 +24,18 @@
 from dcpy.utils import s3, git, versions
 from dcpy.utils.logging import logger
 
+assert (
+    PUBLISHING_BUCKET
+), "'PUBLISHING_BUCKET' must be defined to use edm.publishing connector"
+BUCKET = PUBLISHING_BUCKET
```

Comment:
This is a funny little thing here - mypy is fine with this, but not okay without line 30 (and just using the asserted variable later on)